package.json

```diff
@@ -11,9 +11,10 @@
     "fmt": "prettier --trailing-comma=es5 --write *.js src/*.js examples/*.js test/*.js",
     "serve": "http-server -c-1 -a 127.0.0.1 -o development.html",
     "develop": "concurrently \"npm run build:watch\" \"npm run serve\"",
-    "test": "karma start --single-run --browsers ChromeHeadless karma.conf.js",
-    "test:jest": "jest test.js",
-    "test:watch": "karma start --auto-watch --browsers ChromeHeadless karma.conf.js"
+    "test": "jest",
```

Comment:
Does this mean karma and mocha can be removed as dependencies?

Reply:
yes! i need to clean up good point